sqlitecluster/SQLiteNode.cpp

```diff
@@ -2774,3 +2823,25 @@ void SQLiteNode::_sendStandupResponse(SQLitePeer* peer, const SData& message) {
     }
     _sendToPeer(peer, response);
 }
+
+void SQLiteNode::_dieIfForkedFromCluster() {
+    size_t quorumNodeCount = 0;
+    size_t forkedFullPeerCount = 0;
+    for (const auto& p : _peerList) {
+        if (!p->permaFollower) {
+            forkedFullPeerCount++;
+            if (p->forked) {
+                forkedFullPeerCount++;
+            }
+        }
+    }
+
+    // Am *I* a permaFollower?
```

Comment:
Can you explain here why it counts itself as a quorum node when it is a permafollower please?

Reply:
The condition  is `if (_originalPriority != 0) {` so its counting itself when it is not a permafollower

Reply:
The condition  is `if (_originalPriority != 0) {` so its counting itself when it is not a permafollower